bench_runner/scripts/workflow.py

```diff
@@ -0,0 +1,376 @@
+        "-m",
+        module,
+        *cmd,
+    ]
+
+    if sudo:
+        ld_library_path = os.environ.get("LD_LIBRARY_PATH", "")
+        args = ["sudo", f"LD_LIBRARY_PATH={ld_library_path}"] + args
+
+    print("Running command:", " ".join(args))
+    subprocess.check_call(args)
+
+
+def should_run(
+    force: bool,
+    fork: str,
+    ref: str,
+    machine: str,
+    pystats: bool,
+    flags: list[str],
+    cpython: Path = Path("cpython"),
+    results_dir: Path = Path("results"),
+) -> bool:
+    try:
+        commit_hash = git.get_git_hash(cpython)
+    except subprocess.CalledProcessError:
+        # This will fail if the cpython checkout failed for some reason. Print
+        # a nice error message since the one the checkout itself gives is
+        # totally inscrutable.
+        print("The checkout of cpython failed.", file=sys.stderr)
+        print(f"You specified fork {fork!r} and ref {ref!r}.", file=sys.stderr)
+        print("Are you sure you entered the fork and ref correctly?", file=sys.stderr)
+        # Fail the rest of the workflow
+        sys.exit(1)
+
+    found_result = has_result(
+        results_dir,
+        commit_hash,
+        machine,
+        pystats,
+        flags,
+        benchmark_definitions.get_benchmark_hash(),
+        progress=False,
+    )
+
+    if force:
+        if found_result is not None:
+            for filepath in found_result.filename.parent.iterdir():
+                if filepath.suffix != ".json":
+                    git.remove(results_dir.parent, filepath)
+        should_run = True
+    else:
+        should_run = (machine in ("__really_all", "all")) or found_result is None
+
+    return should_run
+
+
+def checkout_cpython(fork: str, ref: str, cpython: PathLike = Path("cpython")):
+    git.clone(cpython, f"https://github.com/{fork}/cpython.git", branch=ref, depth=50)
+
+
+def checkout_benchmarks():
+    for repo in benchmark_definitions.BENCHMARK_REPOS:
+        git.clone(
+            Path(repo.dirname),
+            repo.url,
+            branch=repo.hash,
+            depth=1,
+        )
+
+
+def compile_unix(cpython: PathLike, flags: list[str], pgo: bool, pystats: bool) -> None:
+    cpython = Path(cpython)
+
+    env = os.environ.copy()
+    if "CLANG" in flags:
+        if sys.platform.startswith("linux"):
+            env["CC"] = util.safe_which("clang-19")
+            env["LLVM_AR"] = util.safe_which("llvm-ar-19")
+            env["LLVM_PROFDATA"] = util.safe_which("llvm-profdata-19")
+        elif sys.platform == "darwin":
+            llvm_prefix = util.get_brew_prefix("llvm")
+            env["CC"] = f"{llvm_prefix}/bin/clang"
+            env["LDFLAGS"] = f"-L{llvm_prefix}/lib"
+            env["CFLAGS"] = f"-I{llvm_prefix}/include"
+
+    if sys.platform == "darwin":
+        openssl_prefix = util.get_brew_prefix("openssl@1.1")
+        env["PKG_CONFIG_PATH"] = f"{openssl_prefix}/lib/pkgconfig"
+
+    args = []
+    if pystats:
+        args.append("--with-pystats")
```

Comment:
I think this should be `--enable-pystats`

Reply:
Thanks, good catch.